File: Atividade_03/nome.sobrenome.py
nome = (input("Digite seu nome: "))
sobrenome = (input("Digite seu sobrenome: "))

print(sobrenome, ',', nome[0], sep='') 

len(nome)
tamanho = len(nome)

ultima_letra = nome[tamanho -1]
print(f"A última letra do seu nome é {ultima_letra}.")

File: Atividade_03/nome.sobrenome.cpp
#include <iostream>
#include <string>

using namespace std;

int main(){
    string nome, sobrenome, tamanho;
    cout << "Digite seu nome: ";
    cin >> nome;
    tamanho = nome.size()-1;
    cout << "Digite seu Sobrenome: ";
    cin >> sobrenome;

    cout << sobrenome << "," << nome[0] << tamanho << endl;
    return 0;
}